js/src/logger.ts

```diff
@@ -2697,23 +2699,33 @@ export async function loadPrompt({
   }
 
   const state = stateArg ?? _globalState;
-
-  await state.login({
-    orgName,
-    apiKey,
-    appUrl,
-    fetch,
-    forceLogin,
-  });
-
-  const args: Record<string, string | undefined> = {
-    project_name: projectName,
-    project_id: projectId,
-    slug,
-    version,
-  };
-
-  const response = await state.apiConn().get_json("v1/prompt", args);
+  let response;
+  try {
+    await state.login({
+      orgName,
+      apiKey,
+      appUrl,
+      fetch,
+      forceLogin,
+    });
+    response = await state.apiConn().get_json("v1/prompt", {
+      project_name: projectName,
+      project_id: projectId,
+      slug,
+      version,
+    });
+  } catch (e) {
+    console.error("Failed to load prompt:", e);
```

Comment:
maybe make this a warning? And explain that you'll try to fall back to the locally cached version?

Also, should we cache them on disk? We used to do this for some stuff, but a fairly common pattern is to create something like `~/.cache/braintrust` (overridable with an env variable, preferrably named like `BRAINTRUST_QUEUE_DROP_EXCEEDING_MAXSIZE`, etc) and save stuff there.

Reply:
Done!